metapool/scripts/seqpro.py

```diff
@@ -61,6 +62,17 @@ def format_preparation_files(run_dir, sample_sheet, output_dir, pipeline):
             df = df.merge(stats.xs(lane, level=1), how='left',
                           on='sample_name')
 
+        # strip qiita_id from project names in sample_project column
+        df['sample_project'] = df['sample_project'].map(
+            lambda x: re.sub(r'_\d+$', r'', x))
+
+        # hardcode center_name to be 'UCSD'
```

Comment:
Isn't this from the sample_sheet? If not, then it means that is hardcoded somewhere else.

Reply:
One moment, I'll confirm where.

Reply:
Confirmed. I removed the assignment before it's written out to disk and changed the 'CENTER_NAME' hardcode to 'UCSD'.